tests/test_data.py

```diff
@@ -744,6 +744,13 @@ def test_providers(self):
                             f"in prefix publication list (appears as {other.title})",
                         )
                         self.assert_publication_identifiers(publication)
+                    # Check provider-specific example handling
+                    if provider.example:
```

Comment:
Is there not already a test that checks there's a `$1` character in the provider's URI format?

I think this added block can be removed